components/deprecated_channel_name/BUILD.gn

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
This should have visibility, otherwise the code owner doesn't prevent anyone from using this

Reply:
also we should explicitly remove it from top level DEPS allowed headers and add it only where needed